vector/src/main/java/im/vector/riotx/features/roomprofile/settings/RoomSettingsAction.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.riotx.features.roomprofile.settings
+
+import im.vector.riotx.core.platform.VectorViewModelAction
+
+sealed class RoomSettingsAction : VectorViewModelAction {
+    data class SetRoomName(val newName: String) : RoomSettingsAction()
```

Comment:
Souldn't be nullable types?

Reply:
Yes, maybe, this is not used for the moment, we will change it when implementing the feature